frontend/src/index.pug

```diff
@@ -77,7 +77,7 @@ html
 
     vuetemplate#v_summary
       #summary.wrapper
-        form.summary-picker.mui-form--inline(v-on:submit.prevent="getFiltered();")
+        form.summary-picker.mui-form--inline(onsubmit="return false;")
```

Comment:
If it does nothing, why we still need this `"return false;"`?

Reply:
`return false;` is to prevent the page from reloading after submitting.

In browsers such as Chrome and Firefox, pressing enter after keying in search value will submit the form. Without `return false;`, the page will reload.

Reply:
The implementation is similar to the filter by glob functionality.

https://github.com/reposense/RepoSense/blob/37af0b469682d4eb73a0e9f2311ee51bc12754db/frontend/src/index.pug#L238-L240